ndcube/utils/wcs.py

```diff
@@ -139,6 +139,10 @@ def _wcs_slicer(wcs, missing_axis, item):
     missing_axis: `list` of `bool`
         Altered missing axis list.  Note the ordering has been reversed to reflect the data
         (numpy) axis ordering convention.
+        
+    extra_coords:
```

Comment:
Perhaps we should rename this ```dropped_coords``` to avoid any confusion with the ```extra_coords``` that may already be attached to the ```NDCube``` object.

Reply:
Yup, agree.